src/main/java/de/tum/in/www1/exerciseapp/web/rest/CourseResource.java

```diff
@@ -34,9 +38,11 @@
     private static final String ENTITY_NAME = "course";
 
     private final CourseService courseService;
+    private final AuthorizationCheckService authCheckService;
 
-    public CourseResource(CourseService courseService) {
+    public CourseResource(CourseService courseService, AuthorizationCheckService authCheckService) {
```

Comment:
@VitaNuova just FYI:
Project is not compiling anymore because unit tests are not compiling anymore. Unit test still use the old constructor, see here: [CourseResourceIntTest](https://github.com/ls1intum/ArTEMiS/blob/develop/src/test/java/de/tum/in/www1/exerciseapp/web/rest/CourseResourceIntTest.java#L93)

I have fixed that on my branch (hd/umleditor) but I'm wondering why the code is compiling on your machine 🤔

Reply:
Thanks for the hint @sockeqwe I fixed it in the develop branch as well with my latest commit.